packages/core/src/types/config.ts

```diff
@@ -15,6 +15,7 @@ export type GeneratorOptions = {
 };
 
 export type MitosisConfig = {
+  commonOptions: { typescript: boolean };
```

Comment:
We want all of this to be optional

```suggestion
  commonOptions?: { typescript?: boolean };
```